extensions/notebook/src/test/book/book.test.ts

```diff
@@ -179,11 +179,12 @@ describe('BooksTreeViewTests', function () {
 				sinon.restore();
 			});
 
-			it('getChildren should return all book nodes when element is undefined', async function (): Promise<void> {
+			it('getChildren should return only root node when element is undefined', async function (): Promise<void> {
```

Comment:
This behavior doesn't seem correct, there can be multiple "root" nodes - one for each book

Reply:
thanks! let me double check.